plugin/ale/cursor.vim

```diff
@@ -72,7 +72,7 @@ function! ale#cursor#TruncatedEcho(message)
 
     try
         " Echo the message truncated to fit without creating a prompt.
-        set shortmess+=T
+        setl shortmess+=T
```

Comment:
Ok after re-checking, the `ProhibitAbbreviationOption` is only for vim options, so no, there is no way to check this kind of error with `vint`.